docs/toggles/application_error_report.rst

```diff
@@ -0,0 +1,17 @@
+Application Error Report
+========================
+This plugin is a useful tool for debugging errors that are preventing mobile users from
```

Comment:
:1st_place_medal: for using the term "plugin". It does appear that is what we use for external docs for toggles.